index.html

```diff
@@ -7,7 +7,15 @@
     <title>Document</title>
     <link rel="stylesheet" href="./style.css">
 </head>
-<body>  
+<body> 
+    <header>
+        <img class="header" src="./img/image1.png" alt="фото">
```

Comment:
Why you need this class ? if you can use header img ? alt="NAME ONLY IN ENGLISH"